scrape_views.py

```diff
@@ -302,3 +323,25 @@ def get_link(show, key, starturl):
         if args.avg:
             print( 'IMDB ratings average plot')
             average_plot(views, average)
+            
+    if not args.epi:
+        epi =""
+    else:
+        epi = args.epi
+        season_num=0
+        episode_num=0
+        if(len(epi)==6):
```

Comment:
This is an exceedingly complex method. We could go for a much simpler solution with this.

Given a string x.
Find the location of s/S => sindex
Find the location of e/E => eindex
```
season_num = int( x[sindex:eindex] )
episode_num = int( x[eindex+1:] ) # within try except
```

This way is much easier. How to find out the location of s/S is trivially searchable.

Reply:
Well Sir now I have updated the code.